package.json

```diff
@@ -136,7 +136,7 @@
     "@types/text-table": "0.2.0",
     "@typescript-eslint/parser": "1.1.0",
     "@zeit/dockerignore": "0.0.4",
-    "@zeit/fun": "0.0.4",
+    "@zeit/fun": "http://zeit-fun-0.0.5.zeit.sh",
```

Comment:
👍, got an error while running `yarn install`:

```
error http://zeit-fun-0.0.5.zeit.sh: Integrity check failed for "@zeit/fun" (computed integrity doesn't match our records, got "sha1-HkLvdBQ3A8MQc519II8IfDwoiWw=")
```

Reply:
Sorry about that @amio. Should be fixed by 7e668cea6a65be7bcf695b9c306520636d7c6a4e.